lisa/base_tools/yum_config_manager.py

```diff
@@ -0,0 +1,49 @@
+from typing import TYPE_CHECKING
+
+from lisa.executable import Tool
+
+if TYPE_CHECKING:
+    from lisa.operating_system import Posix
+
+
+class YumConfigManager(Tool):
+    @property
+    def command(self) -> str:
+        return "yum-config-manager"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def install(self) -> bool:
+        posix_os: Posix = self.node.os  # type: ignore
+        package_name = "yum-utils"
+        posix_os.install_packages(package_name)
```

Comment:
Try to add `# type: ignore` on this line, there is no need on `from lisa.operating_system import Posix` and `posix_os: Posix = self.node.os`

Reply:
update